### PR TITLE
update autobuilds

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updates to autobuilds

### CODE REVIEW

**Review by mcapell — APPROVED** [review]
Looks good to me, thank you!

File: docker-hub/builds/index.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ repositories with Automated Builds configured.
...
+integration workflow where a build that fails its tests doesn't push the built
+image. Automated tests don't push images to the registry on their own. [Learn about automated image testing](automated-testing.md).
 
-Depending on your [plan](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"},
+Depending on your [subscription](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"},
 you may get concurrent builds, which means that `N` autobuilds can be run at the
-same time. `N` is configured by the plan that you subscribe to. Once `N+1` builds
-are running, any additional builds are queued to be run later.
+same time. `N` is configured according to your subscription. Once `N+1` builds
+are running, any additional builds goes into a queue to be run later.
```
Comment (craig-osterhout) [inline docker-hub/builds/index.md:43]:
```suggestion
are running, any additional builds go into a queue to be run later.
```

File: docker-hub/builds/index.md [hunk 2]
```diff
@@ -55,29 +55,31 @@ build an image each time you push new code to your source provider. If you have
 [automated tests](automated-testing.md) configured, the new image is only pushed
 when the tests succeed.
 
-Builds can be added to existing repositories, or added when you create a repository.
+You can add builds to existing repositories, or added when you create a repository.
```
Comment (craig-osterhout) [inline docker-hub/builds/index.md:58]:
```suggestion
You can add builds to existing repositories, or add them when you create a repository.
```

File: docker-hub/builds/index.md [hunk 3]
```diff
@@ -34,16 +34,16 @@ repositories with Automated Builds configured.
...
+Depending on your [subscription](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"},
 you may get concurrent builds, which means that `N` autobuilds can be run at the
-same time. `N` is configured by the plan that you subscribe to. Once `N+1` builds
-are running, any additional builds are queued to be run later.
+same time. `N` is configured according to your subscription. Once `N+1` builds
+are running, any additional builds goes into a queue to be run later.
 
-The total number of pending builds in the queue is limited to 30 and further
-requests will be discarded. The number of concurrent builds for Pro is 5 and
+The maximum number of pending builds in the queue is 30 and discards further
```
Comment (craig-osterhout) [inline docker-hub/builds/index.md:45]:
```suggestion
The maximum number of pending builds in the queue is 30 and Docker Hub discards further
```
I think some subject should be added. I'm not 100% sure it's Docker Hub though.

File: docker-hub/builds/index.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/builds/index.md:43]: Updated, thank you!

File: docker-hub/builds/index.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/builds/index.md:58]: Updated

File: docker-hub/builds/index.md (hunk 3 above)
Comment (jerae-duffin) [reply docker-hub/builds/index.md:45]: Good point. I think it is Hub, I'll confirm with Marc.

Comment (jerae-duffin) [reply docker-hub/builds/index.md:45]: Will update in new commit.

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**